### PR TITLE
[Synapse] Add spark job releated commands based on track2 sdk

### PR DESCRIPTION
**Pull request by sunsw1994**
**Description**  

[Data Plane] -- Add synapse spark job releated commands based on track2 sdk

See details: #14722

**Testing Guide**  
az synapse spark job -h
az synapse spark session -h
az synapse spark statement -h

[commands](https://github.com/sunsw1994/azure-cli/blob/dfb4f6e380bb34f2ca3df89813e0c31a40b85fd5/src/azure-cli/azure/cli/command_modules/synapse/commands.py)

**History Notes**  

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

File: azure-cli2017.pyproj [hunk 1]
```diff
@@ -935,7 +935,13 @@
     <Compile Include="azure-cli\azure\cli\command_modules\storage\_transformers.py" />
     <Compile Include="azure-cli\azure\cli\command_modules\storage\_validators.py" />
     <Compile Include="azure-cli\azure\cli\command_modules\storage\__init__.py" />
+    <Compile Include="azure-cli\azure\cli\command_modules\synapse\operations\spark.py">
+      <SubType>Code</SubType>
+    </Compile>
     <Compile Include="azure-cli\azure\cli\command_modules\synapse\operations\__init__.py" />
+    <Compile Include="azure-cli\azure\cli\command_modules\synapse\util.py">
+      <SubType>Code</SubType>
+    </Compile>
```
Comment (idear1203) [inline azure-cli2017.pyproj:944]: Make them consistent with others?

File: src/azure-cli/azure/cli/command_modules/ams/commands.py [hunk 2]
```diff
@@ -13,9 +13,9 @@
 from ._exception_handler import ams_exception_handler
 
 
-# pylint: disable=line-too-long
+# pylint: disable=line-too-long, disable=too-many-locals, too-many-statements
```
Comment (idear1203) [inline src/azure-cli/azure/cli/command_modules/ams/commands.py:17]: why do you change AMS?

**Comment by yonzhan** [comment]
Synapse

File: src/azure-cli/azure/cli/command_modules/ams/commands.py (hunk 2 above)
Comment (sunsw1994) [reply src/azure-cli/azure/cli/command_modules/ams/commands.py]: Reverted.

File: azure-cli2017.pyproj (hunk 1 above)
Comment (sunsw1994) [reply azure-cli2017.pyproj]: Fixed

**Pull request merged**